doc/ide.md

```diff
@@ -56,6 +56,68 @@ This creates a file named `jvm.vcxproj` in `ide\hotspot-visualstudio`
 subfolder of the build output folder. The file can be opened in Visual Studio
 via `File -> Open -> Project/Solution`.
 
+#### Eclipse CDT
+
+The make system can generate an Eclipse CDT Workspace that enables Eclipse
+indexing for the C and C++ sources throughout the entire codebase, as well as
+registering all common make targets to be runnable from the Eclipse explorer.
+This can be done after configuring by running:
+
+```
+make eclipse-native-env
+```
+
+After this is run, simply open and import the workspace in Eclipse through
+`File -> Import -> Projects from Folder or Archive` and at
+`Import source` click on the directory `ide\eclipse`, which can be
+found in the build output folder.
+
+If this doesn't work, you can also try
+`File -> Import -> Existing Projects into Workspace`
+instead.
+
+Setting up an Eclipse Workspace is relatively lightweight compared to other
+supported IDEs, but requires that your CDT installation has Cross GCC support
+enabled at the moment, even if you aren't cross compiling. The Visual C++
+compiler is, at present, not supported as an indexer.
+
+If desired, you can instead request make to only include indexing support for
+just the Java Virtual Machine instead of the entire native codebase, by running:
+
+```
+make eclipse-hotspot-env
+```
+
+A quick tip for development with Eclipse in general, which also applies to
+indexing the Java sources in the JDK (see below), is to enable dark mode
+before doing so. Trust us, it looks much better than Eclipse's default look
+and feel. ;)
```

Comment:
Does this change how Eclipse functions with our sources in any way, or is it just your opinion that it looks better? If the latter, then I don't think it really belongs in this document.

Reply:
Ah, was hoping that I could leave a personal touch in there. No matter, I'll remove it if required

Reply:
Please remove it.